src/utils/workflow-utils.ts

```diff
@@ -23,6 +23,7 @@ import { ConnectorConfig } from '../../deps/flow/flow';
 
 // This is the soft limit we recommend to users
 export const MAX_BINDINGS = 300;
+export const DEFAULT_UPDATE_WAIT = 500;
```

Comment:
This constant is tightly coupled the `debounce` function. If that is case, it would be helpful to communicate that in the name (e.g., `DEFAULT_DEBOUNCE_WAIT`).

Reply:
I'll update. I considered that but wasn't sure if that would be too specific.